src/ui-config/marketsConfig.tsx

```diff
@@ -545,7 +545,8 @@ export const marketsData: {
       LENDING_POOL: AaveV3Harmony.POOL,
       WETH_GATEWAY: AaveV3Harmony.WETH_GATEWAY,
       WALLET_BALANCE_PROVIDER: AaveV3Harmony.WALLET_BALANCE_PROVIDER,
-      UI_POOL_DATA_PROVIDER: AaveV3Harmony.UI_POOL_DATA_PROVIDER,
+      UI_POOL_DATA_PROVIDER: '0xeC6118C69af50660231108059ab98CD0cF9a6eA1',
```

Comment:
@miguelmtzinf should this be read from address book? or is just to test

Reply:
test, will replace once I am sure it works